content/docs/12-enterprise-version/03-deploying-palette-with-helm.md

```diff
@@ -0,0 +1,326 @@
+
+    <br />
+
+    ```shell
+    tar xzvf path/to-file/spectro-mgmt-helm-charts-X.X.tar.gz 
+    ```
+
+11. Navigate to the **spectro-mgmt-helm-charts-X.X** folder.
+
+    <br />
+
+    ```shell
+    cd spectro-mgmt-helm-charts-X.X
+    ```
+
+12. Review the **values.yaml** .You must populate the parameter `env.rootDomain` to the custom domain you will use for the installation. In addition, add the same `rootDomain` with port `:4222` to the `natsUrl` in the `nats` section of the YAML. Example: `env.rootDomain: my-domain.com:4222`
+
+    All other parameter values are optional and can be reset or changed with a Helm upgrade operation.
+
+    <br />
+    
+ 13. If you wish to use [AWS ACM for SSL Certs](https://docs.aws.amazon.com/acm/latest/userguide/acm-overview.html), instead of the default self-signed certificate generated by the Nginx *ingress controller*, you can add it to the `annotations` under `ingress`.
+
+    <br />
+
+    ```yaml
+    ingress:
+      ingress:
+        # Whether to front NGINX Ingress Controller with a cloud
+        # load balancer (internal == false) or use host network
+        internal: false
+
+        # Default SSL certificate and key for NGINX Ingress Controller (Optional)
+        # A wildcard cert for config.env.rootDomain, e.g., *.myfirstpalette.spectrocloud.com
+        # If left blank, the NGINX ingress controller will generate a self-signed cert (when terminating TLS upstream of ingress-nginx-controller)
+        # certificate: ""
+        # key: ""
+
+        annotations:
+        # AWS example
+        service.beta.kubernetes.io/aws-load-balancer-internal: "true"
+        service.beta.kubernetes.io/aws-load-balancer-backend-protocol: tcp
+        service.beta.kubernetes.io/aws-load-balancer-ssl-cert: "<ARN:ACM>"
+        service.beta.kubernetes.io/aws-load-balancer-ssl-ports: "https"
+        ingressStaticIP: ""
+    ```
+ 
+ 14. Download the kubeconfig file for the EKS cluster. Ensure you can interact with the target cluster. You can validate by issuing a `kubectl` command. For additional guidance, refer to the [kubeconfig file for an Amazon EKS](https://docs.aws.amazon.com/eks/latest/userguide/create-kubeconfig.html) guide.
+
+
+
+15. Install the Helm Chart using the following command. Replace the path in the command to match your local path of the Palette Helm Chart.
+
+  <br />
+
+  ```shell
+  helm install palette /path/to/chart.tgz --file /path/to/values.yaml
+  ```
+
+16. Monitor the deployment using the command below. Palette is ready when the deployments in namespaces `cp-system`, `hubble-system`, `jet-system` , and `ui-system` reach the *Ready* state.
+
+    <br />
+
+    ```shell
+    kubectl get pods --all-namespaces --watch
+    ```
+
+17. Create a DNS record that is mapped to the Palette `ingress-nginx-controller` load balancer. You can use the following command to retrieve the load balancer IP address.
```

Comment:
Should the doc refer to ingress-nginx-controller as a load balancer? Isn't it ingress-nginx-controller OR load balancer? (could be my misunderstanding)